src/notation/internal/notationuiactions.cpp

```diff
@@ -180,16 +180,21 @@ const UiActionList NotationUiActions::m_actions = {
              ),
     UiAction("cut",
              mu::context::UiCtxNotationFocused,
-             QT_TRANSLATE_NOOP("action", "Cut")
+             QT_TRANSLATE_NOOP("action", "Cut"),
+             QT_TRANSLATE_NOOP("action", "Cut"),
+             IconCode::Code::CUT
              ),
     UiAction("copy",
              mu::context::UiCtxNotationFocused,
              QT_TRANSLATE_NOOP("action", "Copy"),
+             QT_TRANSLATE_NOOP("action", "Copy"),
```

Comment:
I really wonder how this could have ever worked before that addition. IMHO it should not even have compiled.

Reply:
Should I create an issue for that?
\

Reply:
There is a constructor that allows us to add an icon without having to add a description
https://github.com/musescore/MuseScore/blob/master/src/framework/ui/uitypes.h#L185

Reply:
There is a constructor that allows us to add an icon without having to add a description
https://github.com/musescore/MuseScore/blob/master/src/framework/ui/uitypes.h#L185
Do we really need descriptions here?

Reply:
Ah, that explains why it worked before.
And in that case there's no need at all for the duplication

Reply:
Ok, I will update this pull request